backend/src/openarchiefbeheer/destruction/tests/e2e/features/test_feature_list_create.py

```diff
@@ -221,3 +224,35 @@ def create_data():
 
             await self.then.this_number_of_zaken_should_be_visible(page, 1)
             await self.then.page_should_contain_text(page, "ZAAK-3")
+
+    async def test_scenario_filter_expired_archive_date(self):
+        @sync_to_async
+        def create_data():
+            today = dateutil.utils.today()
```

Comment:
There is also the `datetime.date.today()` (https://docs.python.org/3/library/datetime.html#datetime.date.today)